src/content/en/fundamentals/media/manipulating/files.md

```diff
@@ -0,0 +1,329 @@
+contains a short list of codecs used for DASH and HLS, and commands needed to
+convert to them.
+
+## File properties
+
+I've unpealed the onion. Yet if you compare what I've done to the list of stated
+goals you'll see that I'm not quite done. Anong the remaining items are bitrate
+and resolution. These properties correlate to the amount of data in a media
+file. It probably goes without saying, but I'm going to say it anyway, that you
+can always lower bitrate and resolution, but you can't increase them. (Well,
+technically you could, but the quality would be bad. Don't do it.)
+
+The first to changing bitrate and resultion is to [display the file
+characteristics](/web/fundamentals/media/manipulating/cheatsheet#display_characteristics)
+and verify that your source file has a higher bitrate or resultion than your
+desired result.
+
+### Bitrate
+
+_Bitrate_ is the maximum number of bits used to encode one second of a stream.
+The more bits used to encode a second of stream, the higher the fidelity.
+
+Unsuprisingly, bitrates for the web are low. The table below compares web
+bitrates with common home video formats. Values are given in megabits per second
+(Mbs).
+
+| Format | Bitrate |
+| ------ | ------- |
+| Blue-ray | 20Mbs |
+| DVD | 6 Mbs |
+| Desktop web video | 2 Mbs |
+| 4G mobile video | 0.7 Mbs |
+| 3G mobile video | 0.35 Mbs |
+| 2G mobile video | ?? |
+
+Which value should I use for video on my web pages? The short answer is at least
+the top three. If you're serving video someplace like India, you'll want to
+include 2G as well. For demonstration purposes, I'm going to target 3G.
+
+In Shaka Packager you set the bitrate by adding the bandwidth option to the
+appropriate stream descriptor. The following, for example, sets the bitrate on
+the video stream, but not the audio stream.
+
+    packager \
+      input=myvideo.mp4,stream=audio,output=myvideo_audio.mp4 \
+      input=myvideo.mp4,stream=video,output=myvideo_video.mp4,bandwidth=350000
+
+In ffmpeg you set the bitrate with the (surprise!) bitrate (`-b`) flag.
+
+    ffmpeg -i glocken.mov -b:v 350K -b:a 350K glocken.mp4
+
+Notice that there are two bitrate flags, -b:a and -b:v. One is for audio and the
+other is for video.
+
+### Resolution
+
+_Resolution_ is the amount of information in a single frame of video, given as
+
+the number of logical pixels in each dimension. [Youtube
+recommends](https://support.google.com/youtube/answer/6375112) the following
+resolutions for video uploads, all in the 16:9 aspect ratio. There's nothing
+specific to Youtube about this list, which makes it a good guide for video on
+the web in general.
+
+| Abbreviation | Dimensions |
+| ------------ | ---------- |
+| 2160p | 3840x2160 |
+| 1440p | 2560x1440 |
+| 1080p | 1920x1080 |
+| 720p | 1280x720 |
+| 480p | 854x480 |
+| 360p | 640x360 |
+| 240p | 426x240 |
+
+Which one do I use? Again, the short answer is all of them. Fortunately, this is
```

Comment:
I'd just make this functional: "You might use one or all of these depending upon your application. Fortunately, this is one of the simplest..."